packages/ember-application/lib/system/engine-instance.js

```diff
@@ -154,6 +159,32 @@ if (isEnabled('ember-application-engines')) {
       setEngineParent(engineInstance, this);
 
       return engineInstance;
+    },
+
+    /**
+      Clone dependencies shared between an engine instance and its parent.
+
+      @private
+      @method cloneParentDependencies
+    */
+    cloneParentDependencies() {
+      let parent = getEngineParent(this);
+
+      [
+        'route:basic',
+        'event_dispatcher:main',
+        P`-bucket-cache:main`,
+        'service:-routing'
+      ].forEach((key) => {
+        this.register(key, parent.resolveRegistration(key));
```

Comment:
I haven't dug in much, but the Travis failures indicate that either parent is undefined in the tests or it doesn't have resolveRegistration...